extensionsGallery-insider.json

```diff
@@ -3090,14 +3090,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.4.0",
-							"lastUpdated": "10/06/2020",
+							"version": "0.5.0",
```

Comment:
Generally it's useful to have insiders released immediately so we can have that channel be testing it as well. Is there any specific reason you'd want to wait until the release happens to update insiders?

Reply:
oops, didn't see the other comments. Disregard =)